DOCS/history-OSI-SPDX.md

```diff
@@ -0,0 +1,87 @@
+# A History of OSI-approved licenses and the SPDX License List
+
+In the first releases of the SPDX License List, it was determined that the SPDX License List should include all licenses that were ever OSI-approved. (A policy that SPDX-legal maintains to this day.) The rationale being that once OSI-approved, always OSI-approved and that even old or "deprecated" (by way of a newer version or other such circumstance) licenses may still appear "in the wild". (Note: the OSI and SPDX use the term "deprecated" in different ways.)
+
+In 2011, OSI [began using the SPDX license identifiers](https://opensource.org/minutes20110608) in their URLs and on the license pages.
+
+The SPDX-legal team sought to accurately identify all OSI-approved licenses and their corresponding text to ensure representation on the SPDX License List. This involved cross-collaboration between SPDX-legal and various OSI representatives, with most communication occurring on the SPDX-legal mailing list, OSI's license-disucss mailing list, or both. John Cowan, Martin Michlmayr, Karl Fogel, and Luis Villa were among the OSI board members who helped Jilayne Lovejoy from SPDX-legal on researching the various issues. 
+
+This page seeks to explain some of the challenges that this reconciliation work encountered, especially where a solution to the challenge may still leave questions for people currently looking at or using both the SPDX License List and the OSI list of approved licenses. Any suggestions that had been made in the past but not implemented or that would aid in clarifiction have been added here as well. 
+
+Since that time, the OSI began more actively reviewing licenses and the website got updated in various ways; and SPDX adopted new policies, such as the license expression syntax. As a result, other issues have appeared in term of "matching" up the licenses that appear on both lists.
+
+This document is an effort to capture the history, as well as acknowledge and identify some issues, which in some cases, may have suggested resolutions.  This list may not be exhaustive. As things get updated, we will try to update this page in a timely manner.
+(last update: July 1, 2024)
```

Comment:
Suggest moving "last update" to the top of the document, under the first header line? Otherwise I'm assuming we'll forget to update it over time...